cvlucaslugarinho.html

```diff
@@ -0,0 +1,27 @@
+<!--LUCAS BEBE-->
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <!--meta http-equiv="X-UA-Compatible" content="IE=edge" (?? use or not use??)-->
+    <meta description="this page is my curriculum vitae as an exercize to learn html">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>LUCAS BEBE CV</title>
+    <meta name="mobile-web-app-capable" content="yes">
+		<meta name="apple-mobile-web-app-capable" content="yes">
+		<meta name="apple-mobile-web-app-status-bar-style" content="black">
+		<meta name="apple-mobile-web-app-title" content="">
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+    </head>
+  <body>
+    <img src="images/20190910_181454_HDR.jpg" alt="furry ass" width="128" height="128">
+    <div class="w3-container">
+      <h2>LUCAS LUGARINHO CV</h2>
```

Comment:
You should be using an H1 since is the most important test in the HTML

Reply:
<3